Demo/Package.swift

```diff
@@ -0,0 +1,4 @@
+// swift-tools-version:5.3
```

Comment:
Is this file needed?

Reply:
It is a sort of hack to exclude that folder from the normal package
"Added an empty (almost) Package.swift to the Demo-folder, that way that folder will be hidden from the main package"

So instead of:
<img width="269" alt="Screen Shot 2021-02-05 at 14 15 20" src="https://user-images.githubusercontent.com/3169203/107038393-bbf76f80-67bc-11eb-84b0-04bb43600353.png">

You get:
<img width="269" alt="Screen Shot 2021-02-05 at 14 13 31" src="https://user-images.githubusercontent.com/3169203/107038435-cade2200-67bc-11eb-926c-f32184adf5ec.png">

Reply:
Found that workaround here: https://forums.swift.org/t/hiding-ignoring-directories-from-xcode-when-opening-swift-packages/35431/6

Reply:
Weird hack, but it definitely makes the overview much easier!